opm/simulators/wells/BlackoilWellModelRestart.cpp

```diff
@@ -260,4 +267,6 @@ loadRestartData(const data::Wells&                 rst_wells,
     }
 }
 
+template class BlackoilWellModelRestart<double>;
```

Comment:
Is there a way to find all the explicit instantiations like this? For when we need to add a float instantiation, for example.

Reply:
there will be a define FLOW_INSTANTIATE_FLOAT that you can grep for.